README.md

```diff
@@ -1,7 +1,123 @@
-#JavaScript Hashes
+# JavaScript "Hashes"
 
 ## Objectives
-+ Create a hash in JS
-+ Access a value from a hash
-+ Add a key-value pair to a hash
-+ Delete a key-value pair from a hash
\ No newline at end of file
++ Explain what a hash in JavaScript is
++ Create an object in JS
++ Access a value from an object
++ Add a key-value pair to an object
++ Delete a key-value pair from an object
++ Iterate over key-value pairs in an object
+
+## Intro 
+
+In JavaScript, all objects are effectively key-value pairs. We're not going to talk about objects in relation to Object Orientation, but objects as hashes. We will eventually get to objects and properties of objects, but right now we're just focused on how to create, manipulate and delete key-value pairs from a "hash".
```

Comment:
maybe add a little more to explain how this is different from Ruby. "JavaScript Objects behave sort of like a cross between Classes and Hashes in Ruby. For now, we're just going to focus on how to use them like a hash."